include/ccf/node/startup_config.h

```diff
@@ -105,6 +105,8 @@ namespace ccf
     std::string service_subject_name = "CN=CCF Service";
     ccf::COSESignaturesConfig cose_signatures;
 
+    std::optional<std::string> sealed_ledger_secret_location;
```

Comment:
These should also be added to `doc/host_config_schema/cchost_config.json`, with a suitable description.